File: JDK11/opentraceability/src/main/java/opentraceability/models/common/sbdhorganization.java
package opentraceability.models.common;

import opentraceability.Constants;
import opentraceability.utility.attributes.OpenTraceabilityAttribute;
import opentraceability.utility.attributes.OpenTraceabilityObjectAttribute;

public class SBDHOrganization {
    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "Identifier")
    public String Identifier = null;

    @OpenTraceabilityObjectAttribute
    @OpenTraceabilityAttribute(Constants.SBDH_NAMESPACE, "ContactInformation")
    public SBDHContact ContactInformation = new SBDHContact();
}

File: Java/OpenTraceability/src/main/kotlin/opentraceability/models/common/sbdhorganization.kt
package opentraceability.models.common

import opentraceability.Constants
import opentraceability.utility.attributes.*

class SBDHOrganization {

    @OpenTraceabilityAttribute(opentraceability.Constants.SBDH_NAMESPACE, "Identifier")
    var Identifier: String = ""


    @OpenTraceabilityObjectAttribute
    @OpenTraceabilityAttribute(opentraceability.Constants.SBDH_NAMESPACE, "ContactInformation")
    var ContactInformation: SBDHContact = SBDHContact()

}
